src/transformers/models/altclip/configuration_altclip.py

```diff
@@ -77,12 +77,8 @@ class AltCLIPTextConfig(PretrainedConfig):
         use_cache (`bool`, *optional*, defaults to `True`):
             Whether or not the model should return the last key/values attentions (not used by all models). Only
             relevant if `config.is_decoder=True`.
-        classifier_dropout (`float`, *optional*):
```

Comment:
`hidden_dropout_prob` and `attention_probs_dropout_prob` are used, but not this one.